### DISCUSSION TITLE
Jump from 0.x.x version to 1.x.x

### DISCUSSION THREAD

**Discussion by freakzlike** [semantic-release/semantic-release#2870] [Q&A] [answered]
Hi,

in one of my projects (https://github.com/freakzlike/i18n-extract) I want to jump from version `0.2.0` to `1.0.0`, but I'm not quite sure how to do so. A feature or breaking commit will only release a `0.3.0` version.

How can I force the version to change to `1.0.0`?

https://github.com/freakzlike/i18n-extract/blob/main/release.config.cjs
```js
module.exports = {
  branches: ['main'],
  plugins: [
    '@semantic-release/commit-analyzer',
    '@semantic-release/release-notes-generator',
    '@semantic-release/changelog',
    '@semantic-release/npm',
    '@semantic-release/github',
    '@semantic-release/git'
  ]
}
```

**Comment by travi** [comment, marked as answer]
please see https://semantic-release.gitbook.io/semantic-release/v/beta/support/faq#can-i-set-the-initial-release-version-of-my-package-to-0.0.1. since semantic-release does not support the alternate versioning rules of major-version-zero, making a commit with a breaking change will result in a v1.0.0

**Reply by freakzlike** [reply]
I made a mistake at the breaking commit. That's why I was not working when I tried. Working now. Thank you 👍
